src/util.rs

```diff
@@ -368,6 +368,59 @@ pub mod test_helpers {
             receipts[0].ra().expect("Balance expected")
         }
     }
+
+    pub fn check_expected_reason_for_opcodes(opcodes: Vec<Opcode>, expected_reason: PanicReason) {
```

Comment:
nit: consider taking `opcodes: &[Opcode]` instead of owned vec so it doesn't have to be cloned.

If we need owned opcodes, then it might be preferable to take an iterator of opcodes because this way you can also pass vecs.

Reply:
created issue #226